wiki/Tournaments/TWC/2020/en.md

```diff
@@ -0,0 +1,225 @@
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | ![][flag_AR] [juankristal](https://osu.ppy.sh/users/443656), ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370) |
+| Map Selector | ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766), ![][flag_JP] [tasuke912](https://osu.ppy.sh/users/2774767) |
+| Commentator | To be Announced...
+| Referee | ![][flag_ES] [Deif](https://osu.ppy.sh/users/318565), ![][flag_DE] [p3n](https://osu.ppy.sh/users/123703), ![][flag_US] [tigereyes144](https://osu.ppy.sh/users/6499811), ![][flag_CL] [WalterToro](https://osu.ppy.sh/users/5281416), ![][flag_GB] [Yazzehh](https://osu.ppy.sh/users/7068973) |
+| Statistician | ![][flag_NZ] [deadbeat](https://osu.ppy.sh/users/128370), ![][flag_DE] [Nwolf](https://osu.ppy.sh/users/1910766) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1018778)
+- [Livestream](https://www.twitch.tv/osulive)
+- **Statistics sheet** will be available shortly after the Live Drawings on Feb 29th (14:00 UTC)
+
+---
+
+## Participants
+
+Participating countries wil be anounced once the Registration phase ends and all submitted registrations have been verified.
+Participating players will be announced once confirmed by their respective country's elected team captain.
+
+---
+
+## Mappools
+
+Mappools will be announced one week prior to the commencement of each stage of the tournament.
+
+---
+
+## Ruleset
+
+### Tournament Rules
+
+1. The osu!taiko World Cup is a country-based team tournament, played on the osu!taiko game mode.
+   - **While this competition is planned as a 2 versus 2 setup, this might change depending on the amount of incoming registrations.**
+2. Beatmap scoring is based on Score V2.
+3. The maps for each round will be announced by the mapset selector in advance on the Sunday before the actual matches take place. Only these will be used during the respective matches.
+   - One map will be given as a tiebreaker map. This map will only be played in case of a tie.
+   - There will also be a [Hidden](/wiki/Game_Modifiers), [HardRock](/wiki/Game_Modifiers), [DoubleTime](/wiki/Game_Modifiers) and FreeMod bracket.
+4. Match schedule will be settled by the Tournament Management (see below).
+5. If no staff or referee is available, the match will be postponed.
+6. Failed players' scores do not get added to the team score.
+   - Reviving and surviving during a map gets considered as passing it.
+7. Use of the Visual Settings to alter background dim or disable map elements like storyboards and skins are allowed.
+8. If the beatmap ends in a draw, the game will be nullified.
+   - If a game is nullified the beatmap will be replayed.
+9. If a player disconnects, they get treated as if they failed the map.
+   - Disconnects within 30 seconds after beatmap begin can be rematched. This is up to the referee's discretion. The played beatmap might be aborted for this.
```

Comment:
wording sounds a bit off. small change could make this better

> Disconnects within 30 seconds of a beatmap's beginning can be remade. This is up to the referee's discretion. The played beatmap might be aborted for this.

Reply:
Tried to simplify